src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -91,7 +92,7 @@ private String getAddress(Principal principal) {
     @GET
     @Path("/eventTypes")
     @RolesAllowed("read")
-    public Multi<EventType> getEventTypes(@BeanParam Query query, @QueryParam("applicationId") UUID applicationId) {
+    public Multi<EventType> getEventTypes(@BeanParam Query query, @QueryParam("applicationId") List<UUID> applicationId) {
```

Comment:
Maybe this should be renamed to "applicationIds" or something to indicate it takes a list?